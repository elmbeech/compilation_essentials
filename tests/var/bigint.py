x = 552921504606846976
print(x)
