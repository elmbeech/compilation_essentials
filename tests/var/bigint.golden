552921504606846976
